docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -0,0 +1,373 @@
+[[_fine_grain_permissions]]
+
+=== Fine grained admin permissions V2
+
+The Fine-Grained Admin Permissions feature allows server administrators to delegate realm management to other administrators, 
+the realm administrators. Different than the regular Role-Based Access Control Mechanism provided through the Global and 
+Realm specific roles (<<_master_realm_access_control, Master realm access control>>), this feature provides a more fine-grained 
+control over how realm resources can be accessed and managed based on a well-defined set of operations that can be performed 
+on them.
+
+By relying on a Policy-Based Access Control, server administrators can define permissions to realm resources such as users, 
+groups, and clients, using different policy types, or access control methods, so that a realm administrator is limited to 
+access a subset of realm resources and their operations.
+
+Here are some key points to understand about fine-grained admin permissions:
+
+* Fine grained admin permissions are only available within <<_per_realm_admin_permissions, dedicated admin consoles>> or 
+  realm-specific admin API and admins defined within those realms. You cannot define cross-realm fine grained permissions.
+* Fine grained admin permissions are used to grant additional permissions. You cannot override the default behavior of the 
+  built-in admin roles.
+* Fine-grained admin permissions V2 do not support federated resources. This is a known limitation under consideration 
+  for future improvement.
```

Comment:
@pedroigor do you think it is sufficient to mention it here withing "key points"? Or would you suggest some changes?

Reply:
Works for me to highlight it here. Thanks.